### ISSUE TITLE
Custom HUD colors revisited

### ISSUE THREAD

**Issue by Rhaedas** [joncage/ed-scout#139]
This is more something to put on the shelf for the future. Nothing wrong with the current implementation of finding the XML colors and duplicating, but there could be more players in the near future using a different method. You probably have seen it, (https://www.reddit.com/r/EliteDangerous/comments/jbfu17/elite_dangerous_hud_mod_edhm_v131_update/) but the fact that it's coming alone with plans for a GUI and even more additions means it could be something to consider when used with this app once enough use it.

How to approach it is the real question, it makes hunting for the XML look easy. Perhaps the GUI will have something to work with, rather than trying to go further in. Could be something to request from the developer, as others might benefit as well from an easy reference?

**Comment by joncage** [comment]
Yeah, I did see that. Definitely one to keep an eye on and, as you say, see if we can collaborate with them...

**Comment by joncage** [comment]
https://github.com/BlueMystical/EDHM_UI/issues/1
